src/util.js

```diff
@@ -80,8 +80,16 @@ Util.isFirefoxAndroid = (function() {
   };
 })();
 
+Util.isR7 = (function() {
```

Comment:
This could be more future-proof by generalizing it as `isLandscapePrimary()` for similar devices in the future that are not R7's. Also some comments on what landscape primary means (orientation off by 90?). Is there any other way of detecting a landscape primary device in the navigator other than user agent parsing?

Reply:
On the one hand, I agree.  On the other hand, I'm not entirely sure whether the behavior observed with the R-7 is correct for a landscape-primary device and is the right basis to generalize.  Are there others you know of that can be used to confirm?

Reply:
Unfortunately no, this configuration for a browser is new and surprising to me :smile:

Reply:
I don't think that `window.screen.orientation.type` is generally implemented either, which seems like the "right" way to indicate and check.

Reply:
What do CSS media queries report for portrait/landscape?

Reply:
not sure, is there a webpage I can load and tell you what it says?

Reply:
Made this quickly: https://heavy-tempo.glitch.me/, just renders the background differently from media queries, but appears that these [don't necessarily correlate to device orientation..](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation).

```
<html>
  <head>
    <style>
      @media screen and (orientation: landscape) {
        body { background-color:red; } }
      @media screen and (orientation: portrait) {
        body { background-color: blue; } }
    </style>
  </head>
  <body>
  </body>
</html>
```

Also it seems that `orientation.type`, even if implemented, is just a string version of the rotation in degrees, so that would change on all devices on every rotation. Does the R7 at least implement orientation.type? Maybe a general getOrientation function would be useful, returning the orientation in degrees, and attempts the orientation type values (again, hoping that landscape-primary devices [which is different than the [landscape-primary value from orientation-type]( https://developer.mozilla.org/en-US/docs/Web/API/Screen/lockOrientation)] implement this)

Reply:
The latest version of the change doesn't mess with window.orientation (which I think is what messed up A-Frame down the line), but simply modifies the interpretation of the gyroscope.

Until we understand more about other landscape-primary devices, and how we can reliably detect them, I think it may be better to keep this scoped to R-7 and expand/generalize in subsequent PRs.